From: Oleksij Rempel <o.rempel@pengutronix.de>
Date: Fri, 23 Sep 2022 12:20:16 +0200
Subject: [PATCH] ARM: boards: Skov-i.MX6: select KSZ8873 switch driver

The board code depends now on the switch driver probe functionality. If
this driver is not enabled, we will make wrong decision.

Signed-off-by: Oleksij Rempel <o.rempel@pengutronix.de>
---
 arch/arm/mach-imx/Kconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/mach-imx/Kconfig b/arch/arm/mach-imx/Kconfig
index 356a91644552..6e4f296055b0 100644
--- a/arch/arm/mach-imx/Kconfig
+++ b/arch/arm/mach-imx/Kconfig
@@ -635,6 +635,8 @@ config MACH_SKOV_IMX6
 	select ARCH_IMX6
 	select ARM_USE_COMPRESSED_DTB
 	select MCI_IMX_ESDHC_PBL
+	select DSA
+	select DRIVER_NET_KSZ8873
 
 endif
 
